src/utils/HTTPTransport.ts

```diff
@@ -0,0 +1,99 @@
+export enum Method {
+  Get = 'Get',
+  Post = 'Post',
+  Put = 'Put',
+  Patch = 'Patch',
+  Delete = 'Delete',
+}
+
+type Options = {
+  method: Method;
+  data?: any;
+};
```

Comment:
Я бы предложил добавить сюда еще headers
```suggestion
type Options = {
  method: Method;
  headers: Record<string, string>;
  data?: any;
};
```